libmscore/note.cpp

```diff
@@ -1530,6 +1530,13 @@ Element* Note::drop(const DropData& data)
                         delete e;
                         return 0;
                         }
+                  if (e->staffIdx() != staffIdx()) {
```

Comment:
What if the note is copied from one staff to the same staff but after a transposition change?